R/board_azure.R

```diff
@@ -194,6 +194,12 @@ board_deparse.pins_board_azure <- function(board, ...) {
   expr(board_azure(!!container, path = !!board$path))
 }
 
+#' @rdname required_pkgs.pins_board
+#' @export
+required_pkgs.pins_board_azure <- function(x, ...) {
+  ellipsis::check_dots_empty()
+  "AzureStor"
```

Comment:
It slightly concerns me that we're now recording this information in two places so it might get out of sync. But I don't see any way to avoid that.